pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl

```diff
@@ -0,0 +1,18 @@
+##########################################
+##                                      ##
+##  AUTOGENERATED CODE -- DO NOT EDIT!  ##
+##                                      ##
+##########################################
+
+testapp "foo" [package="package_foo"]:
+    @version = "0.0.0"
+    @description =:
+        | foo
+
+    /test:
+        GET:
+            | No description.
+            return 200 <: Empty
```

Comment:
this doesn't look right, isn't it usually `... <: 200`?

Reply:
Isn't <: this operator defining the left to be a subset of the right? 
So in that case aren't the two totally different?

Reply:
Yeah I'm kinda confused, what does `... <: `200 mean?

Reply:
Also, just added handling of some extra cases which mean empty:
- Response code without a return Type
- Return type is a response code plus ...
- Return type is ...
- Return type is just Empty
```

    /testResponseCodeEmpty:
        GET:
            | No description.
            return 200 <: Empty

    /testResponseCodeResponse:
        GET:
            | No description.
            return 200

    /testResponseCodeDots:
        GET:
            | No description.
            return 200 <: ...

    /testDots:
        GET:
            | No description.
            return ...

    /testDots:
        GET:
            | No description.
            return Empty

    !type Empty:
        ...

```

Reply:
@cuminandpaprika ah sorry I mean `variable <: 200`, I don't think I've seen `200 <: type` (with type being `Empty` in this case).

Reply:
No worries, fixed up remove support of ... in endpoints!